gammapy/scripts/jupyter.py

```diff
@@ -55,42 +53,47 @@ def cli_jupyter_black(ctx):
         nbformat.write(nb, str(notebook))
 
         # inform
-        print('Jupyter notebook {} painted in black.'.format(str(notebook)))
+        log.info('Jupyter notebook {} painted in black.'.format(str(notebook)))
 
 
 @click.command(name='stripout')
 @click.pass_context
 def cli_jupyter_stripout(ctx):
     """Strip output cells."""
 
-    nblist = build_nblist(ctx)
-
-    for notebook in nblist:
-
+    for notebook in build_nblist(ctx):
         try:
-            subprocess.call("nbstripout '{}'".format(notebook), shell=True)
-            print('Jupyter notebook {} stripped out.'.format(str(notebook)))
+            nb = nbformat.read(str(notebook), as_version=nbformat.NO_CONVERT)
         except Exception as ex:
-            log.error('Error stripping file {}'.format(str(notebook)))
+            log.error('Error parsing file {}'.format(str(notebook)))
             log.error(ex)
+            sys.exit()
+
+        # strip output cells
+        for cellnumber, cell in enumerate(nb.cells):
+            if nb.cells[cellnumber]['cell_type'] == 'code':
+                nb.cells[cellnumber]['outputs'] = []
+
+        # write formatted notebook
+        nbformat.write(nb, str(notebook))
+
+        # inform
+        log.info('Jupyter notebook {} output stripped.'.format(str(notebook)))
 
 
 @click.command(name='execute')
 @click.pass_context
 def cli_jupyter_execute(ctx):
     """Execute jupyter notebook."""
 
-    nblist = build_nblist(ctx)
-
-    for notebook in nblist:
-
+    for notebook in build_nblist(ctx):
         try:
             t = time.time()
             subprocess.call(
                 "jupyter nbconvert --allow-errors --ExecutePreprocessor.timeout=None --ExecutePreprocessor.kernel_name=python3 --to notebook --execute '{}'.format(notebook) --inplace",
```

Comment:
Can you please break this long string?
Suggest something like this:
```
subprocess.call(
    "jupyter nbconvert "
    "--allow-errors "
    "--ExecutePreprocessor.timeout=None "
...
```
Python will just concatenate the strings, and this makes it easy to read and to comment some option out or add code comments in between if something needs an explanation.